addons/common/XEH_postInit.sqf

```diff
@@ -267,8 +267,7 @@ enableCamShake true;
 
 //FUNC(showHud) needs to be refreshed if it was set during mission init
 ["ace_infoDisplayChanged", {
-    GVAR(showHudHash) params ["", "", "_masks"];
-    if (_masks isNotEqualTo []) then {
+    if ((values GVAR(showHudHash)) isNotEqualTo []) then {
```

Comment:
If you want to check for empty, use count or
`hash isEqualTo createHashMap`
Don't grab values, that allocates a new array and copies all values over if its not empty.